mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -197,22 +193,26 @@ More information on module groups can be found {{< docuLinks "/Resources/Documen
 
 ### Macro-Modules {#TutorialMacroModules}
 
-We now like to condense our filter into one single module. You have probably already noticed, that the modules have different colors. Each color represents a special type of modules. The blue modules are called ML-Modules and are responsible for the processing of voxel-images. The brown modules are called macro modules. Macro modules encapsulate a whole network in a single module. You can right-click on one macro-module to open the context menu and select *Show Internal Network* to see the internal network structure. In addition, macro modules enable the implementation of custom functionalities via python scripting.
-
-We now like to condense the modules forming the contour filter into one single macro module. To do that, right-click on the group-title and select *Convert To Local Macro*. Now, you need to select a name for your macro module and finish the creation of the module. With this you create a local macro module. Local macro modules can only be used in your current network.
+You have probably already noticed how the modules differ in color. Each color represents another type of module:
+  
+  * The blue modules are called ML-Modules, they process voxel-images. 
```

Comment:
ML modules, voxel images, macro module, ...

Reply:
Everything in lower case and without hyphens? Okay ^^

Reply:
ML remains upper case :)